provider/gpkg/gpkg_test.go

```diff
@@ -366,14 +352,15 @@ func TestTileFeatures(t *testing.T) {
 				},
 			},
 			layerName: "land2",
-			tile: MockTile{
-				Z:    0,
-				srid: tegola.WebMercator,
-				bufferedExtent: geom.NewExtent(
-					[2]float64{-20026376.39, -20048966.10},
-					[2]float64{20026376.39, 20048966.10},
-				),
-			},
+			tile: slippy.NewTile(0, 0, 0),
+			//MockTile{
+			//	Z:    0,
+			//	srid: tegola.WebMercator,
+			//	bufferedExtent: geom.NewExtent(
+			//		[2]float64{-20026376.39, -20048966.10},
+			//		[2]float64{20026376.39, 20048966.10},
+			//	),
+			//},
```

Comment:
Please remove this code if it no longer needed. If it is needed, put a comment on top as to the reason why.